docs/src/ug/routerules.md

```diff
@@ -62,15 +62,15 @@ They share the same schema but each Input Variable controls the associated route
 - `network_entity_id` accept string value and represent the gateway to be the target of the rule,
 - `description` accept arbitrary string value and give context about the goal of the rule.
 
-#### special values for `network_entity_id`
+#### Special values for `network_entity_id`
 
 For routing rules targeting a Gateway created by the module, `network_entity_id` accepts some special strings to automatically retrieve the gateway ocid.
```

Comment:
In this context, "gateway" should not be capitalised as it's the generic form, not a specific type of gateway.